ring_doorbell/__init__.py

```diff
@@ -388,11 +388,12 @@ def linked_tree(self):
         url = API_URI + LINKED_CHIMES_ENDPOINT.format(self.account_id)
         return self._ring.query(url)
 
-    @property
-    def test_sound(self):
+    def test_sound(self, kind=KIND_DING):
         """Play chime to test sound."""
+        if kind not in CHIME_TEST_SOUND_KINDS:
+            return False
         url = API_URI + TESTSOUND_CHIME_ENDPOINT.format(self.account_id)
-        self._ring.query(url, method='POST')
+        self._ring.query(url, method='POST', extra_params={"kind": kind})
```

Comment:
Shouldn't this be expanded to the "ringtone" the user selected to play?

Reply:
It probably should be. But i'll leave that to a follow-up. Don't have my phone set-up to do the proxy again right now. :)

Reply:
Yes make sense.. You got a good point!!